1_html-css/style.css

```diff
@@ -0,0 +1,359 @@
+    width: 1130px;
+    text-align: center;
+    margin: 60px auto;
+    color: rgb(32, 53, 53);
+    margin-bottom: 80px;
+  
+}
+.box {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    min-height: 400px;
+}
+.card {
+    width: 335px;
+    height: 365px;
+    padding: 20px 35px;
+    border-radius: 20px;
+    background-color: rgb(243, 217, 183);
+    margin: 15px;
+    position: relative;
+    overflow: hidden;
+    text-align: center;
+}
+
+.card i {
+    font-size: 40px;
+    display: block;
+    text-align: center;
+    margin: 25px 0px;
+    color: darkslategrey;
+}
+
+h5 {
+    font-size: 18px;
+    color: #000;
+    margin-bottom: 15px;
+}
+.para p {
+    font-size: 15px;
+    line-height: 27px;
+    margin-bottom: 25px;
+    color: rgb(20, 20, 20);
+}
+
+.card .button {
+    text-decoration: none;
+    font-weight: bold;
+    padding: 9px 18px;
+    border-radius: 30px;
+    transition: 0.4s;
+    border: 2px solid transparent;
+    color: black;
+    background-color: #6b7569;
+}
+.card .button:hover {
+    background-color: transparent;
+    border: 2px solid   #6b7569;
+    cursor: pointer;
+}
+
+.contact {
+    width: 100%;
+    height: 290px;
+    background-color:#d7cbcb;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.contact p {
+    font-size: 30px;
+    font-weight: bold;
+    margin-bottom: 25px;
+    color:  rgb(33, 57, 57);
+}
+
+.contact .button-2 {
+    text-decoration: none;
+    font-weight: bold;
+    padding: 9px 18px;
+    border-radius: 30px;
+    transition: 0.4s;
+    border: 2px solid transparent;
+    color: black;
+    background-color: #6b7569;
+}
+.contact .button-2:hover {
+    background-color: transparent;
+    border: 2px solid   #6b7569;
+    cursor: pointer;
+}
+
+footer {
+    width: 100%;
+    height: 300px;
+    position: relative;
+    background-color:#ede7e7;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+footer p:nth-child(1) {
+    font-size: 30px;
+    color:  rgb(33, 57, 57);
+    margin-bottom: 20px;
+    font-weight: bold;
+}
+footer p:nth-child(2) {
+    font-size: 16px;
+    width: 500px;
+    text-align: center;
+    color: rgb(33, 57, 57);
+    line-height: 26px;
+} 
+
+.social-media {
+    display: flex;
+}
+
+.social-media a {
+    width: 45px;
+    height: 45px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+    margin: 22px 10px;
+    background-color:  rgb(238, 205, 161);
+    text-decoration: none;
+    color: rgb(33, 57, 57);
+    font-size: 20px;
+}
+.social-media a:hover {
+    transform: scale(1.3);
+    transition: 0.3s;
+    background-color: transparent;
+    border: 2px solid   #6b7569;
+    cursor: pointer;
+}
+
+footer p .end {
+    position: absolute;
+    font-size: 8px;
+    bottom: 20px;
+}
+
+/* @media screen and (max-width: 930px) {
```

Comment:
Finish the Media query - and in the future, make sure to no submit commented-out code 🙏🏽